### PR TITLE
Update CCI config to deploy to production

### PR DESCRIPTION
**Pull request by stulees**
## Description
Configure CCI for production deployment

## Motivation and Context
Photos Pitch

## How Has This Been Tested?
Requires merging to `master` branch

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed.
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

**Comment by stulees** [comment]
@olegrise Please review, thanks

**Comment by stulees** [comment]
@olegrise Note, this can not be deployed until this PR is merged/deployed - https://github.com/Rise-Vision/oauth-token-provider/pull/34

File: .circleci/config.yml [hunk 1]
```diff
@@ -80,4 +96,11 @@ workflows:
       - deploy-staging:
           requires:
             - build-image
+      - deploy-production:
+          requires:
+            - deploy-staging
```
Comment (olegrise) [inline .circleci/config.yml:101]: @stulees why `deploy-staging` is required?

Comment (stulees) [reply .circleci/config.yml]: @olegrise It doesn't, I've changed to require `build-image` instead, thanks!

**Review by olegrise — APPROVED** [review]

**Pull request merged**